doc/references.bib

```diff
@@ -2023,4 +2023,19 @@ @Article{Kappenman2021
   publisher = {Elsevier {BV}},
 }
 
+@article{GENOVESE2002870,
+title = {Thresholding of Statistical Maps in Functional Neuroimaging Using the False Discovery Rate},
+journal = {NeuroImage},
+volume = {15},
+number = {4},
+pages = {870-878},
+year = {2002},
+issn = {1053-8119},
+doi = {https://doi.org/10.1006/nimg.2001.1037},
+url = {https://www.sciencedirect.com/science/article/pii/S1053811901910377},
+author = {Christopher R. Genovese and Nicole A. Lazar and Thomas Nichols},
+keywords = {functional neuroimaging, false discovery rate, multiple testing, Bonferroni correction},
+abstract = {Findingobjective and effective thresholds for voxelwise statistics derived from neuroimaging data has been a long-standing problem. With at least one test performed for every voxel in an image, some correction of the thresholds is needed to control the error rates, but standard procedures for multiple hypothesis testing (e.g., Bonferroni) tend to not be sensitive enough to be useful in this context. This paper introduces to the neuroscience literature statistical procedures for controlling the false discovery rate (FDR). Recent theoretical work in statistics suggests that FDR-controlling procedures will be effective for the analysis of neuroimaging data. These procedures operate simultaneously on all voxelwise test statistics to determine which tests should be considered statistically significant. The innovation of the procedures is that they control the expected proportion of the rejected hypotheses that are falsely rejected. We demonstrate this approach using both simulations and functional magnetic resonance imaging data from two simple experiments.}
+}
```

Comment:
The abstract could be deleted to prevent cluttering in the references file. Thoughts, @larsoner @sappelhoff ?

Reply:
agreed